File: handler/src/main/java/cc/vileda/kalfor/handler/KalforProxyRequest.java
package cc.vileda.kalfor.handler;

public class KalforProxyRequest
{
	@SuppressWarnings("WeakerAccess")
	public String path;

	@SuppressWarnings("WeakerAccess")
	public String key;

	@SuppressWarnings("unused")
	public KalforProxyRequest()
	{
	}

	public KalforProxyRequest(final String key, final String path)
	{
		this.key = key;
		this.path = path;
	}
}

File: handler/src/main/kotlin/cc/vileda/kalfor/handler/KalforProxyRequest.kt
package cc.vileda.kalfor.handler

@Suppress("unused")
class KalforProxyRequest {
    var path: String = ""

    var key: String = ""

    constructor(path: String, key: String) {
        this.path = path
        this.key = key
    }

    constructor()
}
